themes/digital.gov/src/scss/new/_communities.scss

```diff
@@ -35,6 +53,12 @@ $communities-of-practice: (
   }
 }
 
+.usa-card.communities {
```

Comment:
We should update the scope of this to make it clear it's part of the featured communities.

Consider updating the class name to something like `.dg-featured-communities .usa-card` or giving it its own class like `.dg-featured-communities__item` or `.dg-featured-communities__card`.